content/guides/javascript/get-program-accounts.md

```diff
@@ -0,0 +1,307 @@
+    {
+      filters: [
+        {
+          dataSize: 165, // number of bytes
+        },
+        {
+          memcmp: {
+            offset: 32, // number of bytes
+            bytes: MY_WALLET_ADDRESS, // base58 encoded string
+          },
+        },
+      ],
+    },
+  );
+
+  console.log(
+    `Found ${accounts.length} token account(s) for wallet ${MY_WALLET_ADDRESS}: `,
+  );
+  accounts.forEach((account, i) => {
+    console.log(
+      `-- Token Account Address ${i + 1}: ${account.pubkey.toString()} --`,
+    );
+    console.log(`Mint: ${account.account.data["parsed"]["info"]["mint"]}`);
+    console.log(
+      `Amount: ${account.account.data["parsed"]["info"]["tokenAmount"]["uiAmount"]}`,
+    );
+  });
+  /*
+    // Output
+
+    Found 2 token account(s) for wallet FriELggez2Dy3phZeHHAdpcoEXkKQVkv6tx3zDtCVP8T: 
+    -- Token Account Address 0:  H12yCcKLHFJFfohkeKiN8v3zgaLnUMwRcnJTyB4igAsy --
+    Mint: CKKDsBT6KiT4GDKs3e39Ue9tDkhuGUKM3cC2a7pmV9YK
+    Amount: 1
+    -- Token Account Address 1:  Et3bNDxe2wP1yE5ao6mMvUByQUHg8nZTndpJNvfKLdCb --
+    Mint: BUGuuhPsHpk8YZrL2GctsCtXGneL1gmT5zYb7eMHZDWf
+    Amount: 3
+  */
+})();
+```
+
+### `dataSlice`
+
+Outside of the two filter parameters, the third most common parameter for
+`getProgramAccounts` is `dataSlice`. Unlike the `filters` parameter, `dataSlice`
+will not reduce the number of accounts returned by a query. Instead, `dataSlice`
+will limit the amount of data for each account.
+
+Much like `memcmp`, `dataSlice` accepts two arguments:
+
+- `offset`: The position (in number of bytes) at which to begin returning
+  account data
+- `length`: The number of bytes which should be returned
+
+`dataSlice` is particularly useful when we run queries on a large dataset but
+don’t actually care about the account data itself. An example of this would be
+if we wanted to find the number of token accounts (i.e. number of token holders)
+for a particular token mint.
+
+```typescript filename="get-program-accounts-data-slice.ts"
+import { TOKEN_PROGRAM_ID } from "@solana/spl-token";
+import { clusterApiUrl, Connection } from "@solana/web3.js";
+
+(async () => {
+  const MY_TOKEN_MINT_ADDRESS = "BUGuuhPsHpk8YZrL2GctsCtXGneL1gmT5zYb7eMHZDWf";
+  const connection = new Connection(clusterApiUrl("devnet"), "confirmed");
+
+  const accounts = await connection.getProgramAccounts(
+    TOKEN_PROGRAM_ID, // new PublicKey("TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA")
+    {
+      dataSlice: {
+        offset: 0, // number of bytes
+        length: 0, // number of bytes
+      },
+      filters: [
+        {
+          dataSize: 165, // number of bytes
+        },
+        {
+          memcmp: {
+            offset: 0, // number of bytes
+            bytes: MY_TOKEN_MINT_ADDRESS, // base58 encoded string
+          },
+        },
+      ],
+    },
+  );
+  console.log(
+    `Found ${accounts.length} token account(s) for mint ${MY_TOKEN_MINT_ADDRESS}`,
+  );
+  console.log(accounts);
+
+  /*
```

Comment:
Same re: output. This will make it consistent with the rest of `developer-content`